### PR TITLE
Feat/add mising code block elements to playbooks strategies.rst

### PR DESCRIPTION
**Pull request by hereje**
##### SUMMARY
    feat: (docs): Add missing code-block elements on playbooks_strategies.rst
    
    ** add code-block elements on user guide/playbooks_strategies.rst file
    so that translation processes keep code as is
    
    - hacktoberfest 2021
    
    Fixes: #75911

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
user guide/ playbooks_strategies.rst

##### ADDITIONAL INFORMATION
```
There is no need of additional information
```

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_startnstep.rst [hunk 1]
```diff
@@ -11,7 +11,9 @@ When you are testing new plays or debugging playbooks, you may need to run the s
 start-at-task
 -------------
 
-To start executing your playbook at a particular task (usually the task that failed on the previous run), use the ``--start-at-task`` option::
+To start executing your playbook at a particular task (usually the task that failed on the previous run), use the ``--start-at-task`` option:
```
Comment (Akasurde) [inline docs/docsite/rst/user_guide/playbooks_startnstep.rst]: Please remove this file as this is already modified here - https://github.com/ansible/ansible/pull/75967

**Review by Akasurde — CHANGES_REQUESTED** [review]
Please remove this file as this is already modified here - https://github.com/ansible/ansible/pull/75967

File: docs/docsite/rst/user_guide/playbooks_startnstep.rst (hunk 1 above)
Comment (hereje) [reply docs/docsite/rst/user_guide/playbooks_startnstep.rst]: Done. File does not belong to this change

**Comment by ariordan-redhat** [comment]
Closing this PR: Issue 75911 has already been resolved with a merged PR.

@hereje Thank you for taking the time to contribute.

**Pull request closed without merging**